ramalama/cli.py

```diff
@@ -602,7 +603,10 @@ def pull_cli(args):
     if matching_files:
         return matching_files[0]
 
-    return model.pull(args)
+    try:
+        model.pull(args)
+    except ParseError as ex:
+        perror(f"Failed to pull '{model.model_name}:{model.model_tag}' : {ex}")
```

Comment:
We still want error raised, so the command exits with non 0 status.

Reply:
Ups, you are right. Moved the except block to the cli's main function and use an `EINVAL` as exit code... doesn't really fit, but I couldn't find a better match.